packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -428,27 +382,22 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
           env: loadedEnv?.parsed ?? config.injectableEdgeEnv?.parsed ?? {},
           flags: [],
           clientVersion: config.clientVersion,
+          engineVersion: config.engineVersion,
           previewFeatures: this._previewFeatures,
           activeProvider: config.activeProvider,
-          inlineSchema: config.inlineSchema,
-          inlineDatasources: config.inlineDatasources,
-          inlineSchemaHash: config.inlineSchemaHash,
+          inlineSchema: config.inlineSchema ?? '',
+          overrideDatasources: options.datasources ?? {},
+          inlineDatasources: config.inlineDatasources ?? {},
+          inlineSchemaHash: config.inlineSchemaHash ?? '',
```

Comment:
Node: These should not require any defaults, they are required and not nullable.